src/api/app/models/concerns/staging_project.rb

```diff
@@ -87,19 +85,32 @@ def broken_packages
     @broken_packages
   end
 
+  def missing_reviews_for_classified_requests(request, reviews)
+    @missing_reviews_of_st_project ||= []
+    current_missing_reviews = []
+
+    reviews.each do |review|
+      next if review.state == :accepted || review.by_project == name
+      result = extract_missing_reviews(request, review)
+      current_missing_reviews << result
+      @missing_reviews_of_st_project << result
+    end
+    current_missing_reviews
+  end
+
   def missing_reviews
-    return @missing_reviews if @missing_reviews
+    return @missing_reviews_of_st_project if @missing_reviews_of_st_project
 
-    @missing_reviews = []
+    @missing_reviews_of_st_project = []
 
-    Review.includes(:bs_request).where(bs_request_id: staged_requests.select(:id)).where.not(state: :accepted).find_each do |review|
+    Review.includes(bs_request: [:bs_request_actions]).where(bs_request_id: staged_requests.select(:id)).where.not(state: :accepted).find_each do |review|
       # We skip reviews for the staging project since these reviews are used
       # by the openSUSE release tools _after_ the overall_state switched to
       # 'accepted'.
       next if review.by_project == name
```

Comment:
@krauselukas, would be possible to add this condition as part of the query? Something like:

```
Review.includes(bs_request: [:bs_request_actions]).where.not(by_project: name)....
```

Reply:
Completly overlooked that point, thanks @DavidKang